CDCgov/trusted-intermediary
# Patient Resource - Contact Section

# Add Contact Section to Patient Resource

This PR adds the code that is responsible for adding the `Contact` section in the `Patient` resource. The `Contact` section gets populated by  extracting the values of the following sections under `Patient` resource:
- `extension.humanName`
- `address`
- `telecom`

In addition to this, the `relationship` sub-section is added to the `Contact` section.

### Testing

1. Fire up the application
2. Use postman to send FHIR message (`oml_message.json` located in the examples folder)
3. Make sure the out put message has the `Contact` section and that it has the following sub-sections:
- name
- relationship
- address
- telecom
4. Make sure all tests pass

#491 


## Checklist

- [x] I have added logging where useful (with appropriate log level)
- [x] I have added JavaDocs where required
